### PR TITLE
[DCSFUS-40] Querying The Transportation Database

### PR DESCRIPTION
**Pull request by evansaizik**
I was able to import and query the required data from the transportation database.
Screenshot 2022-08-23 at 12 11 12
Screenshot 2022-08-23 at 12 11 28

### CODE REVIEW

File: submissions/isaiahUsiholo/database/transportQuery.sql [hunk 1]
```diff
@@ -0,0 +1,34 @@
+SELECT * FROM registered_passengers;
+
+
+-- the total number of people who survived = 290;
```
Comment (ToluEnny24) [inline submissions/isaiahUsiholo/database/transportQuery.sql:4]: Your query is wrong, maybe you imported less than 891 people. Also select * from..... should be together with this new query

File: submissions/isaiahUsiholo/database/transportQuery.sql [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+SELECT passenger_status, COUNT(passenger_status) FROM accidents
+WHERE passenger_status = 'survived';
+
+-- the total number of people who did not survive = 424;
```
Comment (ToluEnny24) [inline submissions/isaiahUsiholo/database/transportQuery.sql:8]: Same as above

**Review by ToluEnny24 — COMMENTED** [review]
Hi @evansaizik well done, but you need to reimport your data base

**Review by ToluEnny24 — CHANGES_REQUESTED** [review]
Please effect the requested changes

**Review by Mayowa2020 — CHANGES_REQUESTED** [review]
Well done @evansaizik ! Your code is not running as it should. I tried running the test on your code and it kept failing. Try and reach out to me on Slack, let's work this out together.

**Review by Mayowa2020 — APPROVED** [review]
Well done @evansaizik ! Approved...

**Pull request merged**